prototype_1/homemade_salinity_code.ino

```diff
@@ -0,0 +1,137 @@
+#include <SoftwareSerial.h>
+#define rxpin 2
+#define txpin 3
+
+
+SoftwareSerial myserial(rxpin, txpin);
+
+
+String inputstring = "";
+String sensorstring = ""; 
+boolean input_stringcomplete = false;
+boolean sensor_stringcomplete = false;
+
+int sensorPin = A5;
```

Comment:
I know this is a thing I do too, but the best practice is define or const for variables.